### DISCUSSION TITLE
Can't find SPAM models

### DISCUSSION THREAD

**Discussion by HdeV999** [brainvisa/brainvisa.github.io#122] [Q&A] [answered]
Hi, 
I'm running a dev installation of BrainVISA (v 5.2.0) and I would like to access the 3D visualization of SPAM models. But I can't find them... When I try to run the morphologist-viewers-sulci-Anatomist Show Descriptive Model and click on the database icon, nothing shows up (and when I dig in ~/casa_distro/brainvisa-opensource-master/build/share/brainvisa-share-5.2/models I can't find much either).
How and/or where can I find the SPAM models? 
Many thanks!

**Comment by denisri** [comment, marked as answer]
Hi,
SPAM model data are are not part of the code sources, so they have to be downloaded separately, yes.
There is a process for that in brainvisa: `Morphologist/Sulci/Recognition/SPAM models installation`. You just have to run it with the default parameters.
Denis

**Reply by HdeV999** [reply]
Great, thanks!
